nservicebus/endpoints/index.md

```diff
@@ -1,14 +1,18 @@
 ---
-title: Endpoints
-summary: Describe the concepts of Endpoint and Endpoint Instance.
-reviewed: 2019-07-18
+title: Endpoints and endpoint instances
+summary: Defines the concepts of endpoint and endpoint instance.
+reviewed: 2021-06-29
 component: Core
 related:
  - samples/endpoint-configuration
 redirects:
 - nservicebus/endpoint
 ---
 
-An _endpoint_ is a logical entity that communicates with other endpoints via [_messaging_](/nservicebus/messaging). Each endpoint has an identifying name and contains a collection of [_message handlers_](/nservicebus/handlers/) and [_sagas_](/nservicebus/sagas/). An endpoint can be deployed to a number of machines and environments. Each deployment of an endpoint is an instance. Each endpoint instance processes messages from an input queue which contains messages for the endpoint to process.
+An _endpoint_ is a logical component that communicates with other components using [_messages_](/nservicebus/messaging). Each endpoint has an identifying name, contains a collection of [_message handlers_](/nservicebus/handlers/) and/or [_sagas_](/nservicebus/sagas/), and is deployed to a given _environment_ (e.g. development, testing, production). 
```

Comment:
I think it may be better to use _component_ rather than _entity_ here because _entity_ may risk drawing the reader toward confusion with more specific definitions of the term, such as that defined by DDD, or even Entity Framework. For that reason, _component_ feels more general.